resources/web/style/list.pcss

```diff
@@ -44,8 +44,8 @@
     & > p + div {
       &.orderedlist,
       &.itemizedlist {
-        margin-top: -0.9em;
+        margin-top: 1em;
```

Comment:
I'm curious what would happen if you took this out entirely. I can see why a negative margin could cause problems, but the best CSS rule is the one that isn't even needed.

Reply:
Agreed. Removing it entirely in dev tools looks fine (in Chrome at least).

Reply:
Remove the `margin-top` only, or delete everything from lines 43 to 51?

Reply:
You'd be able to get rid of L41-L50... but only if we're OK with the larger gap shown in my comment [below](https://github.com/elastic/docs/pull/2285#pullrequestreview-806502427).

Reply:
Remove any empty blocks/parent-blocks. So probably lines 41-50 (not the overall closing brace, though). I'd be curious to see how this looks after existing `dd` items, or list items that contain nested paragraphs and divs.